k8s/values.yml

```diff
@@ -1,16 +1,14 @@
-replicaCount: 1
-
-image:
-  repository: pennlabs/image-name
-  tag: latest
-  pullPolicy: IfNotPresent
-
-service:
-  port: 80
-
-ingress:
-  enabled: true
-  domain: example.com
-
-secrets:
-  secretEnv: example
+deploy_version: 0.1.1
+image_tag: latest
+
+applications:
+  - name: worker
+    image: pennlabs/<image_name>
```

Comment:
Replace `<image_name>` with `image-name` so the init script can update the values automaticaly.

Reply:
ah thanks. wrong dash